images/runtime/php/template.Dockerfile

```diff
@@ -22,13 +22,13 @@ RUN ln -s /opt/startupcmdgen/startupcmdgen /usr/local/bin/oryx
 RUN rm -rf /tmp/oryx
 
 # Temporarily configuring and installing gd in php images, need to move to php base image building
-RUN set -eux; \
-    if [[ $PHP_VERSION == 7.4.* ]]; then \
-		docker-php-ext-configure gd --with-freetype --with-jpeg \
-        && PHP_OPENSSL=yes docker-php-ext-configure imap --with-kerberos --with-imap-ssl ; \
-    else \
-		docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr \
-        && docker-php-ext-configure imap --with-kerberos --with-imap-ssl ; \
-    fi \
-    ; \
-    docker-php-ext-install gd
+#RUN set -eux; \
```

Comment:
not clear as to why these have been commented out?

Reply:
this will be removed now as I moved this to php base images